build.rs

```diff
@@ -1,4 +1,58 @@
+use std::env;
```

Comment:
We're using this quite a lot we may want to have a `blueprint-build-utils` crate in `gadget` that can just expose this for different contract_dirs and be reused.